secret/secrets.go

```diff
@@ -0,0 +1 @@
+package secret
```

Comment:
we already have https://github.com/reddit/baseplate.go/blob/master/secrets/doc.go so I don't think we need this file.

Reply:
Purely for my own learning, what does having this (or the other) empty file accomplish?